src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -844,6 +844,7 @@ public with sharing class CRLP_RollupUI_SVC {
         List<Map<String, String>> fieldObjectList = new List<Map<String, String>>();
 
         Map<String, Schema.DescribeFieldResult> objectFields = UTIL_Describe.getAllFieldsDescribe(objectName);
+        // TODO: consider putting this into a map instead of a set
```

Comment:
Out of curiosity, why change to a map?

Reply:
This was a change that Beth had made above to search specific objects for specific fields. IsDeleted I guess would need to be checked every time, but for the other fields do they match more than one object?

Reply:
I see. The only ones unique to Opportunity are the last 5 fields. The first 3 can appear in all objects.